app/mailers/customer_mailer.rb

```diff
@@ -108,9 +108,6 @@ def partial_refund(email, link_id, purchase_id, refund_amount_cents_usd, refund_
   def send_to_kindle(kindle_email, product_file_id)
     product_file = ProductFile.find(product_file_id)
 
-    # Set creator to nil if the product is deleted
-    creator = product_file.link.try(:user)
-
```

Comment:
This is to fix the rubocop violation introduced in https://github.com/antiwork/gumroad/pull/84